pkg/bindinfo/binding_cache.go

```diff
@@ -86,9 +111,34 @@ func (fbc *fuzzyBindingCache) FuzzyMatchingBinding(sctx sessionctx.Context, fuzz
 					break
 				}
 			}
+		} else {
+			missingSQLDigest = append(missingSQLDigest, sqlDigest)
 		}
 	}
-	return
+	return matchedBinding, leastWildcards, isMatched, missingSQLDigest
+}
+
+func (fbc *fuzzyBindingCache) loadFromStore(missingSQLDigest []string) {
+	for _, sqlDigest := range missingSQLDigest {
+		bindings, err := fbc.loadBindingFromStorageFunc(sqlDigest)
+		if err != nil {
+			logutil.BgLogger().Warn("loadBindingFromStorageFunc binding failed", zap.String("sqlDigest", sqlDigest), zap.Error(err))
+			continue
+		}
+		// put binding into the cache
+		oldBinding := fbc.GetBinding(sqlDigest)
+		newBindings := removeDeletedBindings(merge(oldBinding, bindings))
+		if len(newBindings) > 0 {
+			err = fbc.SetBinding(sqlDigest, newBindings)
+			if err != nil {
+				// When the memory capacity of bing_cache is not enough,
+				// there will be some memory-related errors in multiple places.
+				// Only needs to be handled once.
+				logutil.BgLogger().Warn("BindHandle.Update", zap.String("category", "sql-bind"), zap.Error(err))
+			}
+		}
+		updateMetrics(metrics.ScopeGlobal, oldBinding, fbc.GetBinding(sqlDigest), true)
```

Comment:
This metrics is useless now, how about removing it here to make our code simple?

Reply:
removed